Add azure link (#11757)
dotnet/docs: docs/core/docker/intro-net-docker.md

* [Azure App Service](https://azure.microsoft.com/services/app-service/containers/)\
Deploy web apps or APIs using Linux containers in a PaaS environment.

* [Azure Container Instances](https://azure.microsoft.com/services/container-instances/)\

> Develop apps fast without managing virtual machines or having to learn new tools—it's just your application, in a container, running in the cloud.

Author: ```suggestion
Host your container in the cloud without any higher-level management services.
```
